sdk/identity/azure-identity/src/token_credential_impl.cpp

```diff
@@ -274,77 +279,87 @@ AccessToken TokenCredentialImpl::ParseToken(
     }
   }
 
-  if (expiresOnPropertyName.empty())
+  std::vector<std::string> nonEmptyExpiresOnPropertyNames;
+  std::copy_if(
+      expiresOnPropertyNames.begin(),
+      expiresOnPropertyNames.end(),
+      std::back_inserter(nonEmptyExpiresOnPropertyNames),
+      [](auto const& propertyName) { return !propertyName.empty(); });
+
+  if (nonEmptyExpiresOnPropertyNames.empty())
   {
     // 'expires_in' is undefined, 'expires_on' is not expected.
```

Comment:
Does this comment need updating? I don't understand what it's saying.

Reply:
`'expires_in'` is relative expiration, `'expires_on'` is absolute expiration. The comment is still valid.